README.md

```diff
@@ -1,3 +1,55 @@
-# README
+# Respond to an Employment Tribunal Claim
+[![Build Status](https://travis-ci.org/ministryofjustice/et3.svg?branch=develop)](https://travis-ci.org/ministryofjustice/et3)
 
-This app is currently in development and not ready for use or deployment. It will replace the current online version of the ET3 form.
\ No newline at end of file
+Digital ET3 form within the in-house ET service, replacing a third party supplier.
+
+## Getting Started
+This is a Ruby on Rails app which uses [GOV.UK Elements](https://github.com/alphagov/govuk_elements) and sits within the ET service.
+The form submits to the [ET API](https://github.com/ministryofjustice/et_api) which can only be done via the [ET Full System](https://github.com/ministryofjustice/et-full-system).
```

Comment:
In the full system stuff, I try to say that people don't have to use it - because its docker and some people don't like docker - so its up to them if they want to setup a full system themselves.  Plus with these isolated apps, it isn't as necessary to run everything anyway depends on what part you are working on obviously.  May be worth saying something similar or just refer to the full system on how to get it all working.

Reply:
I have amended the "can only be done" line to `The form submits to the [ET API](https://github.com/ministryofjustice/et_api) which has been configured, via Docker, within the [ET Full System](https://github.com/ministryofjustice/et-full-system).`

I think your second point is covered already:
`Running this app will enable development of the app itself, with its test suite mocking the final submission.`
Is that OK?